packages/mongo/package.js

```diff
@@ -88,6 +89,7 @@ Package.onTest(function (api) {
   // XXX test order dependency: the allow_tests "partial allow" test
   // fails if it is run before mongo_livedata_tests.
   api.addFiles('mongo_livedata_tests.js', ['client', 'server']);
+  api.addFiles('mongo_driver_tests.js', ['server']);
```

Comment:
As mentioned above, please move this test into `mongo_livedata_tests.js`.

Reply:
Hello!

I was found additional problems If adding `Decimal128` type this way.
Can't add support to `minimongo`. I'm try using npm package `bson`
to add new EJSON type, but different imports leaves problems.

I will try to make it works, and then change PR code with your remarks.

Reply:
There is still one problem with decimal types. There is `decimal.js` or `big.js` libraries
to make calculations with decimal precision. Maybe should make direct maping to one of these,
instead using `bson` type, which can make only toString conversion.

Need some time to investigate if it possible (use `Decimal128` in `mongodb` and say `big.js` in `Meteor` code).

Reply:
Maybe I should add separate package to this PR to support Decimal128 type, something like `mongo-id`.

Reply:
Not found implemetation of IEEE 754-2008 decimal128 in javascript, but there is [decimal.js](https://github.com/MikeMcl/decimal.js/), which can be a workaround.

Need to add tests and will make commit in a day or two.

Reply:
Hello! I'm removed tests from `mongo` package completely, instead will add tests to new `mongo-decimal` package today.